core/comments/rendered_workspace_comment.ts

```diff
@@ -208,8 +208,15 @@ export class RenderedWorkspaceComment
   private startGesture(e: PointerEvent) {
     const gesture = this.workspace.getGesture(e);
     if (gesture) {
-      gesture.handleCommentStart(e, this);
-      this.workspace.getLayerManager()?.append(this, layers.BLOCK);
+      const textArea = this.view.getTextArea();
```

Comment:
Can you do this with `isTargetInput` from `browser_events.ts` instead?

Reply:
Ooh, yes! Much nicer, thank you!